validation/validation_test.go

```diff
@@ -145,11 +145,13 @@ func (s *MySuite) TestValidateStep(c *C) {
 	valErr := specVal.validateStep(myStep)
 
 	c.Assert(valErr, Not(Equals), nil)
-	c.Assert(valErr.Error(), Equals, "foo.spec:3 Step implementation not found => 'my step'"+"\n"+
-		"Suggestion : "+"\n"+
-		"@Step(\"my step\")"+"\n"+
-		"public void implementation(){"+"\n\t"+
-		"// your code here...\n}")
+	c.Assert(valErr.Error(), Equals, "foo.spec:3 Step implementation not found => 'my step'")
+	c.Assert(valErr.(StepValidationError).Suggestion(), Equals, "One or more errors were due to : Step implementation not found => 'my step'\t\n"+
+		"Suggestion : \n"+
+		"@Step(\"my step\")\n"+
```

Comment:
We should have the suggestion indented so that it looks something like this:

```
Suggestion :
    @Step("my step")
    public void implementation(){
        // your code here...
    }
```